lib/events/filesessions/filestream_test.go

```diff
@@ -128,14 +128,6 @@ func TestCompleteUpload(t *testing.T) {
 				createPart(t, handler, upload, int64(5), []byte("withreservation"))
 			},
 		},
-		{
-			desc:            "OnlyReservation",
-			expectedContent: []byte{},
-			partsFunc: func(t *testing.T, handler *Handler, upload *events.StreamUpload) {
-				createPart(t, handler, upload, int64(1), []byte{})
-				createPart(t, handler, upload, int64(2), []byte{})
-			},
-		},
```

Comment:
@rosstimothy FYI I had to remove this test as it was basically ensuring that we _do_ upload empty uploads if we find an empty upload instead of discarding it.